ibis/omniscidb/ddl.py

```diff
@@ -605,21 +622,108 @@ def _format_schema_element(name, tp, nullable):
     )
 
 
-class InsertPandas(OmniSciDBDML):
-    """Insert Data from Pandas class."""
+class Insert(OmniSciDBDML):
+    """Insert class."""
```

Comment:
Try to use comments that add some value. This comment is just adding noise. And there is some useful information that you can add here, like whether this class is expected to be used directly, or is it only a base class for the other two...